waypoints:
- name: waypoint_0
  joint_positions: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
- name: waypoint_1
  joint_positions: [0.5, -0.3, 0.8, 0.0, 0.5, 0.0]
- name: waypoint_2
  joint_positions: [1.0, -0.5, 1.2, 0.1, 0.7, 0.2]
